### DISCUSSION TITLE
CDNjs not picking up latest version

### DISCUSSION THREAD

**Discussion by heyesr** [cdnjs/cdnjs#14262] [General] [answered]
Hi there,

I've just released a new version of my software - RGraph 6.19 -  but CDNjs doesn't appear to be picking up the new release. The Git Repository is tagged - https://github.com/heyesr/rgraph/tree/6.19 (and branched at this tag too fwiw) but so far nada. Do I just need to be more patient for CDNjs to mirror the file?

Thanks very much!

Richard

**Comment by MattIPv4** [comment, marked as answer]
Moved to https://github.com/cdnjs/cdnjs/issues/14263
